### ISSUE TITLE
segmentation fault Calculate Weight matrix

### ISSUE THREAD

**Issue by hvasbath** [pyrocko/kite#117]
On python 3.11. ubuntu 20.04:
I guess thats again pyqtgraph related? At some point they decided to change all their input float variables to int? 
```
INFO:SceneIO/Gamma:Using Lat/Lon reference
Traceback (most recent call last):
  File "/home/vasyurhm/Software/kite/kite/spool/spool.py", line 205, in progressStarted
    self.progress.setMaximum(maximum)
TypeError: setMaximum(self, maximum: int): argument 1 has unexpected type 'float'
```

**Comment by emolch** [comment]
Should be fixed with 200972bb5880a28e64fc04c845f339f925600fcc

Newer Qt versions check types on various calls and don't automatically cast anymore.

**Comment by emolch** [comment]
But that's probably not causing the segfault, right?
